storage/data-plane/src/test/java/com/microsoft/azure/storage/Samples.java

```diff
@@ -47,6 +36,16 @@
 import java.util.logging.Logger;
 
 public class Samples {
+
+    //Samples only run in Live/Record mode.
+    void checkMode() {
+        String testMode = System.getenv("AZURE_TEST_MODE");
+        if(testMode == null){
+            testMode = "PlAYBACK";
```

Comment:
PLAYBACK (instead of PlAYBACK). Should be a constant?

Reply:
Extracted to constant.